src/js/deckard.js

```diff
@@ -215,12 +215,13 @@
         $html.addClass(device.classes.join(' '));
     };
 
-    $window
-        .on(orientationEvent, function() {
-            $.extend($, orientation());
-            $html.toggleClass('portrait', $.orientation.portrait);
-            $html.toggleClass('landscape', $.orientation.landscape);
-        });
+    var updateOrientation = function() {
+        $.extend($, orientation());
+        $html.toggleClass('portrait', $.orientation.portrait);
+        $html.toggleClass('landscape', $.orientation.landscape);
```

Comment:
Agree. Dependability outweighs DOM touches though in this case. The `toggleClass` function can be a major source of errors where, if you don't use the second param to force the state, it gets into a situation where your logic is reversed.